package.json

```diff
@@ -20,7 +20,9 @@
     "test:ci": "npm run lint && npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
     "bench": "branchcmp -r 2 -g -s \"npm run benchmark\"",
     "benchmark": "npx concurrently -k -s first \"node ./examples/simple.js\" \"npx autocannon -c 100 -d 5 -p 10 localhost:3000/\"",
-    "license-checker": "license-checker --production --onlyAllow=\"MIT;ISC;BSD-3-Clause;BSD-2-Clause\""
+    "license-checker": "license-checker --production --onlyAllow=\"MIT;ISC;BSD-3-Clause;BSD-2-Clause\"",
+    "markdownlint": "markdownlint '**/*.md'",
```

Comment:
This tool is really annoying. I can't find any documentation about what options it accepts in the configuration dotfile to determine if the target file list can be specified there instead of repeated throughout our code base.

As an example of what I'm looking for, https://node-tap.org/docs/configuring/ clearly shows how to generate a default configuration file. It also shows that `--files` can be added to a configuration like so:

```yaml
files:
	- index.test.js
	- 'test/**/*.test.js'
```

Do you know if this same thing can be done with the Markdown lint tool?

Reply:
I'm not sure, I'm sorry you are annoyed, I can close this PR, the purpose was to improve document quality but the amount of grief it seems to cause you does not warrant it.

Reply:
I'm not against the PR. I'm just frustrated with the tool's lack of transparency. As best I can tell, `markdownlint-cli` is the thing that reads the configuration and kicks off the linter. But that tool's readme doesn't provide a list, or a link to a list, of all possible configuration options.

Reply:
That's true.

I've filed:
https://github.com/igorshubovych/markdownlint-cli/issues/158
https://github.com/igorshubovych/markdownlint-cli/issues/159

maybe see what they say and can decide if the tool will be worth fastify maintainers' while.

Reply:
I've ported to `markdownlint-cli2` PTAL and let me know your preference!

The files we want to lint still need to be specified every time it is invoked though.

Reply:
The openjs foundation uses this tool [remark](https://www.npmjs.com/package/remark) that seems more documented to lint MD files